app/assets/stylesheets/static_pages.scss

```diff
@@ -502,6 +502,10 @@
   margin-bottom: 0;
 }
 
+.alert-none {
```

Comment:
yes thats the I want the name to be none to signify that the flash gets hidden 😄